### ISSUE TITLE
Randomisations - implement spatially structured randomisation

### ISSUE THREAD

**Issue by GoogleCodeExporter** [shawnlaffan/biodiverse#76]
```
It is possible to build a spatially structured randomisation, where labels
are assigned randomly to groups based on spatial location such that there
is greater structure in the randomly generated patterns.  

See Laffan and Crisp (2003) for an example
(http://www3.interscience.wiley.com/journal/118882020/abstract ), although
there are others out there.  


```

Original issue reported on code.google.com by `shawnlaffan` on 30 Jan 2010 at 11:13

**Comment by shawnlaffan** [comment]
Label allocation tracking needs issue #558 to be resolved so it works when a subset spatial condition is specified.

**Comment by shawnlaffan** [comment]
Changes were merged in commit 487c152c8a064e77a6d52cb2164db19b63a1150b
